change/@fluentui-react-charting-594b823b-95eb-4e54-968b-d87bc609a723.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fetching theme attributes from tokens",
```

Comment:
This PR should not target master. It should be targeted to a charting-v9 branch.

Reply:
Done. Made a separate branch for v9 under fluentui